nservicebus/tools/migrate-to-native-delivery.md

```diff
@@ -0,0 +1,161 @@
+                        -t|--target <targetConnectionString>
+                        -c|--cutofftime <cutofftime>
+                        --source <source>
+                        --tablename <tablename>
+                        --dialect <sqlDialect>
+                        [--endpoint] <endpointName>
+                        [--allendpoints]
+```
+
+#### Running a migration
+
+Migrating from RavenDB persistence
+
+```
+dotnet migrate-timeouts ravendb
+                        -t|--target <targetConnectionString>
+                        -c|--cutofftime <cutofftime>
+                        --serverUrl <serverUrl>
+                        --databaseName <databaseName>
+                        [--prefix] <prefix>
+                        [--ravenVersion] <ravenVersion>
+                        [--endpoint] <endpointName>
+                        [--allendpoints]
+                        [-a|--abort]
+
+```
+
+Migrating from Sql persistence
+
+```
+dotnet migrate-timeouts preview sqlp
+                        -t|--target <targetConnectionString>
+                        -c|--cutofftime <cutofftime>
+                        --source <source>
+                        --tablename <tablename>
+                        --dialect <sqlDialect>
+                        [--endpoint] <endpointName>
+                        [--allendpoints]
+                        [-a|--abort]
+
+```
+
+Aborting a migration
+
+```
+dotnet migrate-timeouts <repeat previous parameters>
+                        [-a|--abort]
+
+```
+
+## How the tool works
+
+Before actually migrating any timeouts, it's highly recommended to use the preview option.
+This option will perform the following actions:
+ - check that the tool can connect to the storage
+ - check that the tool can connect to the target transport
+ - verify that the necessary topology is in place in the target transport to support native delayed delivery
+ - list all the endpoints for which the tool can detect timeouts
+ - calculate the amount of timeouts to migrate per endpoint
+ - validate if there are timeouts the tool is [unable to migrate](migrate-to-native-delivery.md#limitations)
+
+Once this information has been reviewed, the migration process can be initialized.
+Even though the tool supports migrating all endpoints connected to the persistence at once, it is highly suggested to migrate endpoint by endpoint, especially for critical endpoints. Even when selecting the -allendpoints options, the tool will conduct an endpoint-by-endpoint migration behind the scenes.
+
+## Limitations
+
+As documented in the [RabbitMQ transport](/transports/rabbitmq/delayed-delivery.md), the maximum delay value of a timeout is 8,5 years. If the migration tool encounters any timeouts that have delivery times set beyond that, it won't be possible to migrate that endpoint.
+
+If the tool presents endpoints that are not part of the system when running the --analyze option, it might be that an endpoint was renamed at some point.
```

Comment:
```suggestion
If the tool presents endpoints that are not part of the system when running the `--analyze` option, it might be that an endpoint was renamed at some point.
```

Is it `analyze` or `preview`?